lib/mongoid/scroll/cursor.rb

```diff
@@ -31,6 +35,13 @@ def from_record(record, options)
           cursor.tiebreak_id = record['_id']
           cursor
         end
+
+        def from_base64(str)
```

Comment:
It'll be the caller responsibility to either:
- encrypt/sign the Base64 string before making it public to be sure that it's not modified
- perform a check after deserializing the cursor to make sure that `Mongoid::Scroll::Cursor` params are allowed to sort (`field_name`, `direction`, etc...)